docs/en/query_language/system.md

```diff
@@ -0,0 +1,33 @@
+# SYSTEM Queries {#query_language-system}
+
+- [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
+- [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
+- [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)
+
+## Managing Distributed Tables {#query_language-system-distributed}
+
+ClickHouse can manage [distributed](../operations/table_engines/distributed.md) tables. When a user inserts data into such table, ClickHouse creates a queue of the data which should be sent to servers of the cluster, then asynchronously sends them. You can control the processing of queue by using the requests [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [FLUSH DISTRIBUTED](#query_language-system-flush-distributed) and [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends).
```

Comment:
This should mention `insert_distributed_sync` setting since asynchronous sends are just the default.

Reply:
I'll mention It, but I cannot link the setting because it is not described yet.